### ISSUE TITLE
Random MultilayerGraph generation should be a utility function

### ISSUE THREAD

**Issue by mschauer** [JuliaGraphs/MultilayerGraphs.jl#22]
Goes a bit against the grain to make this task something the constructor does.

https://github.com/InPhyT/MultilayerGraphs.jl/blob/bcfcebfb74ddc2ef77664cb0eb9045307106136e/src/multilayerdigraph.jl#L21

**Comment by InterdisciplinaryPhysicsTeam** [comment]
Hi @mschauer,

Thanks for this issue!

Generally speaking, we think that delegating the instantiation of random graphs to a constructor may lead to better consistency in the ecosystem. After all, if we just substituted the arguments `min_edges` and `max_edges` with `ne`  (the number of required random edges) we would have something very close (the closest possible?) to what `Graphs.jl` already does for its concrete types `SimpleGraph` and `SimpleDiGraphs`. They infact support random graph generation via `SimpleGraph(nv,ne)` and `SimpleDiGraph(nv,ne)`.

 [Here](https://github.com/JuliaGraphs/Graphs.jl/issues/165) we better explain what we mean (see bullet point 2. : it is meant mainly for "non-wrapper" packages, but we tried to stick to that as much as we could).

By the way just noticed that the docstring should be updated, it has `graph_type::Type{<: AbstractGraph}` instead of `graph_types::Vector{DataType}`. We have just fixed it.

CC: @pitmonticone , @ClaudMor
